src/views/Game/PlayerCards.tsx

```diff
@@ -352,13 +356,18 @@ export function PlayerCard({
                         })}
                     </div>
                 )}
-                <NumCapturesText
-                    score={score}
-                    color={color}
-                    zen_mode={zen_mode}
-                    hidden={show_points && !estimating_score}
-                />
-                {!show_points && <div className="komi">{komiString(score.komi)}</div>}
+                {(!show_points || estimating_score) && (
+                    <NumCapturesText
+                        score={score}
+                        color={color}
+                        zen_mode={zen_mode}
+                        hidden={show_points && !estimating_score}
+                    />
+                )}
+                {rules && <div className="rules">{rules}</div>}
+                {!show_points && !!score.komi && (
```

Comment:
Checking for `score.komi` here, because otherwise the blank komi `<div/>` takes up space in the black player's panel.